.github/workflows/integration-test.yaml

```diff
@@ -240,6 +244,8 @@ jobs:
       run: |
        tests/scripts/minikube.sh helm
        tests/scripts/helm.sh up
+       sed -i "s|deviceFilter:  ''|deviceFilter: $(lsblk|awk '/14G/ {print $1}'| head -1)|g" tests/framework/installer/ceph_manifests.go
+       sed -i "s|for i := 0; i < 30; i++ {|for i := 0; i < 60; i++ {|g" tests/framework/utils/k8s_helper.go
```

Comment:
why not just edit `tests/framework/utils/k8s_helper.go` directly?

Reply:
Done.